proxies/shared/partials/Partial.Flows.CreateMessageBatchEndpoint.xml

```diff
@@ -29,6 +33,6 @@
         </Step>
     </Response>
     <Condition>
-        (proxy.pathsuffix MatchesPath "/v1/message-batches") and (request.verb = "POST")
+        proxy.pathsuffix MatchesPath "/v1/message-batches"
```

Comment:
This condition seems wrong, this flow should only execute on a `POST` - if it executes on other verbs then the API is broken.